docs/pipeline/security/index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "Security"
+sidebar_position: 3
+sidebar_label: "Security"
+---
+
+This section contains information relating to pipeline security.
```

Comment:
Could you add the cards here? Or do we not like cards now @mscwilson ? The page just looks a little empty.

Reply:
does this page add much? no info and only one child page